pipeline/config-diff.sh

```diff
@@ -51,6 +53,18 @@ function redact_config_dir {
         fi
         redact_file "$1/src/kayobe-config/$item" "$reference"
     done
+
+    # replace vaulted files with md5sum of the vaulted file
+    for item in "${KAYOBE_CONFIG_VAULTED_FILES_PATHS[@]}"; do
+	# skip if file doesn't exist
+	if [ -f "$1/src/kayobe-config/$item"]; then
+	    if [ "$2" != "" ]; then
+                md5sum "$2/src/kayobe-config/$item" > "$1/src/kayobe-config/$item"
```

Comment:
So the second argument to this function is the kayobe-config on the target branch (the one that you are going to merge into). So I think this would always give the same md5 for source and target files. I've simplified the logic and fixed the style issues that were causing CI to fail.